inkscape/svg2shenzhen/export.py

```diff
@@ -221,15 +218,15 @@ class Svg2ShenzhenExport(inkex.Effect):
     def __init__(self):
         """init the effect library and get options from gui"""
         inkex.Effect.__init__(self)
-        self.OptionParser.add_option("--path", action="store", type="string", dest="path", default="~/", help="")
-        self.OptionParser.add_option('-f', '--filetype', action='store', type='string', dest='filetype', default='jpeg', help='Exported file type')
-        self.OptionParser.add_option("--crop", action="store", type="inkbool", dest="crop", default=False)
-        self.OptionParser.add_option("--dpi", action="store", type="float", dest="dpi", default=600)
-        self.OptionParser.add_option("--threshold", action="store", type="float", dest="threshold", default=128.0)
-        self.OptionParser.add_option("--openfactory", action="store", type="inkbool", dest="openfactory", default="true")
-        self.OptionParser.add_option("--openkicad", action="store", type="inkbool", dest="openkicad", default="true")
-        self.OptionParser.add_option("--autoflatten", action="store", type="inkbool", dest="autoflatten", default="true")
-        self.OptionParser.add_option("--debug", action="store", type="inkbool", dest="debug", default=False)
+        self.arg_parser.add_argument("--path", type=str, dest="path", default="~/", help="")
```

Comment:
You can use the `add_arguments` method here and clean this up a bit.
```
def add_arguments(self, pars):
    pars.add_arguments("--path", default="~/")
```

You don't have to specify dest if dest is the same as the flag, and type=str is the default.

Reply:
Done